pkg/acsengine/defaults.go

```diff
@@ -83,6 +83,14 @@ var (
 		ImageVersion:   "latest",
 	}
 
+	//DefaultCentOSImageConfig is the OpenShift on Centos distribution.
+	DefaultCentOSImageConfig = AzureOSImageConfig{
+		ImageOffer:     "CentOS",
+		ImageSku:       "7.3",
+		ImagePublisher: "OpenLogic",
+		ImageVersion:   "latest",
```

Comment:
We should version these images I believe.

@pweil- @jim-minter

Reply:
What's this for? We'll be building our own CentOS images, hopefully very shortly.

Reply:
I obviously don't understand what's going on with our published images. @kwoodson pointed me to this image and I thought it was one we published. Do you know how can I use our already existing published rhel image? It will need to be added in acs-engine if we want to use it in the ARM template.

Reply:
We will use something like this once we have the correct images published but it is probably too early now.  I also think we'll probably need to cue based on orchestrator version in some fashion so that we can differentiate between image versions since that directly corresponds with the OpenShift binary version.

Reply:
Hey @sylus, I've heard that non-OpenLogic azure CentOS images are in the pipeline, still a little way out, but it's a different team to the folks here.  You'll have to ask via CentOS channels.